python3: can't open file '/root/projects/Bubbles-payload/funpumper/fun_purger_loop': [Errno 2] No such file or directory
  File "/root/projects/Bubbles-payload/funpumper/fun_purger_loop.py", line 34
    pass pass
    ^
IndentationError: expected an indented block after 'try' statement on line 32
